application/frontend/controllers/MethodController.php

```diff
@@ -114,8 +114,8 @@ public function actionCreate()
 
         $request = \Yii::$app->request;
 
-        $value = $request->post('value');
-        if ($value === null) {
+        $value = trim($request->post('value'));
```

Comment:
trim(null) behaviour has changed in PHP 8, as far as I recall. It is safer to null coalesce to empty string before trimming.